### PR TITLE
gh-1207 Add property to override the image on the login page

### LINKED ISSUE #1207: Customise Image On Login Screen
The image used on the login screen is always the default image. This can be slightly confusing when we override the image using the property `spring.boot.admin.ui.brand`.

So can we either have the image on the login screen use the image set in the brand property or have a new property exposed that allows us to override the image in the login page?

### PR DESCRIPTION
**Pull request by michaelmcfadyensky**
resolves #1207

### CODE REVIEW

File: spring-boot-admin-server-ui/src/main/java/de/codecentric/boot/admin/server/ui/config/AdminServerUiProperties.java [hunk 1]
```diff
@@ -49,6 +49,11 @@
      */
     private String templateLocation = CLASSPATH_RESOURCE_LOCATIONS[0];
 
+    /**
+     * Icon used as default image on login page
```
Comment (joshiste) [inline spring-boot-admin-server-ui/src/main/java/de/codecentric/boot/admin/server/ui/config/AdminServerUiProperties.java]:
```suggestion
     * Icon used as image on login page
```

Comment (michaelmcfadyensky) [reply spring-boot-admin-server-ui/src/main/java/de/codecentric/boot/admin/server/ui/config/AdminServerUiProperties.java]: updated

**Comment by michaelmcfadyensky** [comment]
Updated PR with documentation for new property.

**Comment by michaelmcfadyensky** [comment]
With this new property and the title property now existing, could the "brand" property have a new default ie.

`String brand = "<span>${title}</span>";`

This is outside of the scope of this PR but something to be considered.

**Comment by joshiste** [comment]
merged with 9180032a500a577d5b6fa447ec26264f07992f96

**Pull request closed without merging**